client/src/errors.rs

```diff
@@ -20,6 +21,7 @@ impl fmt::Display for SignalClientError {
             Self::RegistrationError(err) => format!("{err}"),
             Self::LoginError(err) => format!("{err}"),
             Self::SendMessageError(err) => format!("{err}"),
+            Self::DatabaseError(err) => format!("{err}"),
```

Comment:
Isnt this just
```
            _err => format!("{_err}"),
```

Reply:
Tror ikke det er muligt, det er værdien vi vil printe, og ikke type konstruktøren